Module/main.js

```diff
@@ -545,4 +552,11 @@ Hooks.once("init", () => {
       button.insertAfter(html.find("#game-details"));
     }
   });
+
+  game.settings.register("bug-reporter", "contactInfo", {
+    scope: "client",
+    config: false,
+    default: "abc",
```

Comment:
debug code?

Reply:
Wait, don't you have to register the setting? Or are you referencing the abc? The abc is definitely debug.

Reply:
yeah I was specifically referencing the `abc` :P